examples/piano.rs

```diff
@@ -147,24 +147,32 @@ fn handle_midi_input(
     query: Query<(Entity, &Key)>,
 ) {
     for data in midi_events.read() {
-        let [_, index, _value] = data.message.msg;
-        let off = index % 12;
-        let oct = index.overflowing_div(12).0;
-        let key_str = KEY_RANGE.iter().nth(off.into()).unwrap();
-
-        if data.message.is_note_on() {
-            for (entity, key) in query.iter() {
-                if key.key_val.eq(&format!("{}{}", key_str, oct).to_string()) {
-                    commands.entity(entity).insert(PressedKey);
+        match data.message {
+            OwnedLiveEvent::Midi {
+                message: MidiMessage::NoteOn { key, .. },
```

Comment:
I noticed in the piano example the NoteOff was not working, works with this:
```suggestion
                message: MidiMessage::NoteOn { key, .. } | MidiMessage::NoteOff { key, .. },
```